archive.php

```diff
@@ -33,6 +33,9 @@
 } else if ( is_post_type_archive() ) {
 	$context['title'] = post_type_archive_title( '', false );
 	array_unshift( $templates, 'archive-' . get_post_type() . '.twig' );
+} else if ( is_tax('p4-page-type') ) {
```

Comment:
Not sure about this hardcoded string... Should I use a constant here? something else?